### PR TITLE
Navbar Component

### PR DESCRIPTION
**Pull request by btzr-io**
## Todo
- [x] Move test data -> [test/data.js](https://github.com/buttercomponents/butter-base-components/blob/master/test/data.js)
- [x] Add layout title
- [ ] Add tooltips
- [x] Integrate goBack button
- [ ] Tidy up

https://github.com/buttercomponents/butter-base-components/blob/master/test/data.js

### CODE REVIEW

File: src/components/toolbar/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    onClick() {
+        let {props, state} = this
+        props.toogle && this.setState((prev) => ({active: !!!prev.active}));
+        props.action.bind(this)
+    }
+
+    render () {
+        let {props, state} = this
+        let classList = Styles([style.button, state.active && style.active, state.update && style.update])
```
Comment (xaiki) [inline src/components/toolbar/index.js:29]: those `&&` don't look necesary, they can be handled in `Styles()`

**Review by xaiki — CHANGES_REQUESTED** [review]
looks good, one minor nitpick

File: src/components/toolbar/index.js (hunk 1 above)
Comment (btzr-io) [reply src/components/toolbar/index.js:29]: What / how ??  😕

Comment (btzr-io) [reply src/components/toolbar/index.js:29]: `Styles()` is for filter / add multiple classes, 
not sure how to handle / validate component `state`

Comment (xaiki) [reply src/components/toolbar/index.js:29]: duh my bad, i read too quick

**Pull request merged**